qadence/backends/braket/backend.py

```diff
@@ -157,9 +166,16 @@ def expectation(
         observable: list[ConvertedObservable] | ConvertedObservable,
         param_values: dict[str, Tensor] = {},
         state: Tensor | None = None,
-        protocol: Measurements | None = None,
+        measurement: Measurements | None = None,
+        noise: Noise | None = None,
         endianness: Endianness = Endianness.BIG,
     ) -> Tensor:
+        # Noise is ignored if measurement protocol is not provided.
+        if noise is not None and measurement is None:
+            logger.warning(
+                f"Errors of type {noise} are not implemented for expectation yet. "
+                "This is ignored for now."
```

Comment:
I would say: "for exact expectation". It is implemented in case of shot-based expectation.